src/CoreText/CTFontManager.cs

```diff
@@ -107,15 +114,17 @@ public static bool IsFontSupported (NSUrl url)
 			return CTFontManagerIsSupportedFont (url.Handle);
 		}
 #elif !XAMCORE_3_0
-#if !NET
-		[Obsolete ("API not available on iOS, it will always return false.")]
+#if NET
+		[UnsupportedOSPlatform ("macos10.6")]
+#if MONOMAC
+		[Obsolete ("Starting with macos10.6.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
+		[UnsupportedOSPlatform ("ios")]
+#else
 		[Deprecated (PlatformName.MacOSX, 10, 6)]
 		[Unavailable (PlatformName.iOS)]
-#else
-		[UnsupportedOSPlatform ("ios")]
-		[UnsupportedOSPlatform ("macos10.6")]
-		[Obsolete ("This API will always return false.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
 #endif
+		[Obsolete ("API not available on iOS, it will always return false.")]
```

Comment:
This guy is outside the #if/else right? does it even make sense to have this attr inside the else?

Reply:
I think it does since it is inside the `#elif !XAMCORE_3_0` block

Reply:
Yes, this was an original Obsolete, not one I created.